content/en/blog/_posts/2024-04-03-windows-ops-readiness.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog
+title: "Introducing the Windows Operational Readiness Specification"
+date: 2024-04-03
+slug: intro-windows-ops-readiness
+---
+
+**Authors:** Jay Vyas (Tesla), Amim Knabben (Broadcom), and Tatenda Zifudzi (AWS)
+
+
+Since Windows support [graduated to stable](/blog/2019/03/25/kubernetes-1-14-release-announcement/)
+with Kubernetes 1.14 in 2019, the capability to run Windows workloads has been much
+appreciated by the end user community. The level of and availability of Windows workload
+support has consistently been a major differentiator for Kubernetes distributions used by
+large enterprises. However, with more Windows workloads being migrated to Kubernetes
+and new Windows features being continuously released, it became challenging to test
+Windows worker nodes in an effective and standardized way.
+
+The Kubernetes project values the ability to certify conformance without requiring a 
+closed-source license for a certified distribution or service that has no intention 
+of offering Windows.
+
+Some notable examples brought to the attention of SIG Windows were:
+
+- An issue with load balancer source address ranges functionality not operating correctly on
+  Windows nodes, detailed in a GitHub issue:
+  [kubernetes/kubernetes#120033](https://github.com/kubernetes/kubernetes/issues/120033).
+- Reports of functionality issues with Windows features, such as
+  “[GMSA](https://learn.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) isn’t working, why?”,
+  discussed in [microsoft/Windows-Containers#44](https://github.com/microsoft/Windows-Containers/issues/44).
+- Challenges developing networking policy tests that could objectively evaluate
+  Container Network Interface (CNI) plugins across different operating system configurations,
+  as discussed in [kubernetes/kubernetes#97751](https://github.com/kubernetes/kubernetes/issues/97751).
+
+SIG Windows therefore recognized the need for a tailored solution to ensure Windows
+nodes' operational readiness *before* their deployment into production environments.
+Thus, the idea to develop a [Windows Operational Readiness Specification](https://kep.k8s.io/2578)
+was born.
+
+## Can’t we just run the official Conformance tests?
+
+The Kubernetes project contains a set of [conformance tests](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/conformance.yaml), 
```

Comment:
Let's link instead to one of:
- https://www.cncf.io/training/certification/software-conformance/#how
- https://kubernetes.io/docs/setup/best-practices/node-conformance/